### PR TITLE
add deploy infra

### PR DESCRIPTION
**Pull request by jamesgorrie**
Adding the deploy infra to wellcomecollections.org.
We can now deploy artifacts created by circle.

The instructions are in the [README](https://github.com/wellcometrust/wellcomecollection.org/blob/3a35283f7e27df3011d2940418530f5e1d60ff63/infra/README.md).

There is still a bit [to do](https://github.com/wellcometrust/wellcomecollection.org/blob/3a35283f7e27df3011d2940418530f5e1d60ff63/infra/TODO.md) (not an exhaustive list) - but for now we have a ELB with content serving, so great.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
@alfraser86

**Comment by jamesgorrie** [comment]
@wellcometrust/web-platform

File: infra/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Deploy
+
+You'll need to be sure to get setup with you `tfvars` from a friendly developer near you.
```
Comment (davidpmccormick) [inline infra/README.md]: Nitpick typo *your?

Comment (jamesgorrie) [inline infra/README.md]: Not sure I spotted the typo but I changed it as it was a rubbish sentence anyway.

**Pull request merged**